### PR TITLE
League of Legends Skin Command

### LINKED ISSUE #1053: League of Legends Skin Command
A command that shows information about a League of Legends skin. Extra points if we link to the respective SkinSpotlights video.

### PR DESCRIPTION
**Pull request by dpaiv0**
+ Closes #1053 
image
image

### CODE REVIEW

File: src/commands/games/leagueoflegends/skin.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        type: 'string', full: true, missingError: 'commands:leagueoflegends.subcommands.skin.noSkin'
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, skin) {
+    channel.startTyping()
+    const embed = new SwitchbladeEmbed(author)
+    try {
+      const split = skin.split('|')
```
Comment (pedrofracassi) [inline src/commands/games/leagueoflegends/skin.js]: what is this even???

that's ugly af

Comment (dpaiv0) [reply src/commands/games/leagueoflegends/skin.js]: how do you think that should be made?
1. there are skins that change the name of the original champ (e.g. Beemo)
2. skins that come after the champions name aren't going to work (e.g. True Damage Senna Prestige Edition)

Comment (dpaiv0) [reply src/commands/games/leagueoflegends/skin.js]: i do have an idea, and i'll work that 2nite if i can get the install working

Comment (pedrofracassi) [reply src/commands/games/leagueoflegends/skin.js]: Use fuze!

Comment (dpaiv0) [reply src/commands/games/leagueoflegends/skin.js]: ok thats a better idea

Comment (dpaiv0) [reply src/commands/games/leagueoflegends/skin.js]: there was no need in using fuse, but it now works!

**Comment by dpaiv0** [comment]
i'm dumb, i forgot that i needed fuse.js again

**Comment by dpaiv0** [comment]
+ standard isnt ok

**Comment by dpaiv0** [comment]
## RFR

File: src/apis/LeagueOfLegends.js [hunk 2]
```diff
@@ -8,17 +8,36 @@ module.exports = class LeagueOfLegends extends APIWrapper {
...
+      })
+
+    await this.loadSkins()
+    return this
+  }
+
+  async loadSkins () {
+    const champions = await this.fetchChampions()
+
+    for (var i in champions) {
```
Comment (fernando457829) [inline src/apis/LeagueOfLegends.js]: Use `const` instead `var`.

**Comment by Doges** [comment]
btw, the YouTube thingy looks dope af

**Pull request merged**